js/app/modules/selection/staticArticles.js

```diff
@@ -0,0 +1,34 @@
+/* exported StaticArticles */
+
+// Copyright 2017 Endless Mobile, Inc.
+
+const Eknc = imports.gi.EosKnowledgeContent;
+const GLib = imports.gi.GLib;
+
+const Module = imports.app.interfaces.module;
+const Xapian = imports.app.modules.selection.xapian;
+
+/**
+ * Class: StaticArticles
+ *
+ * Retrieves articles tagged with EknStaticTag
+ */
+const StaticArticles = new Module.Class({
```

Comment:
We really need https://phabricator.endlessm.com/T13175 so we can stop the proliferation of selections, and have this just be an instance of Filter.Tag or something like that.